easybuild/tools/filetools.py

```diff
@@ -700,9 +700,17 @@ def adjust_permissions(name, permissionBits, add=True, onlyfiles=False, onlydirs
         for root, dirs, files in os.walk(name):
             paths = []
             if not onlydirs:
-                paths += files
+                if not skip_symlinks:
```

Comment:
Thinking about it, it may actually make sense to switch that too (If we are skipping symlinks and if it *is* a link, issue a debug message; otherwise do it)